src/ResizeSensor.js

```diff
@@ -101,7 +101,7 @@
             if (element.currentStyle) {
                 return element.currentStyle[prop];
             }
-            if (window.getComputedStyle) {
+            if (window.getComputedStyle && window.getComputedStyle(element, null)) {
```

Comment:
shouldn't `window.getComputedStyle(element, null)` be cached in a variable as it's quite expensive to calculate?